handler.go

```diff
@@ -39,6 +39,7 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer cancel()
 
 	logFrameHeader(r)
+	log.Println("Request started")
```

Comment:
I don't think `log.Println` flushes, is this a problem? (I can see it might be, if the customer's trying to figure out why there code isn't running, but maybe it's out of scope for this change?)

Reply:
It appears to flush, I think the problem was in Python fdk iirc.